test/views/HowManyBroughtIntoUkFromSmallProducersViewSpec.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import config.FrontendAppConfig
+import forms.HowManyAsAContractPackerFormProvider
+import models.{CheckMode, LitresInBands, NormalMode}
+import play.api.data.Form
+import play.api.i18n.Messages
+import play.api.mvc.Request
+import play.api.test.FakeRequest
+import play.twirl.api.HtmlFormat
+import views.helpers.{LitresSpecHelper, ViewSpecHelper}
+import views.html.HowManyBroughtIntoTheUKFromSmallProducersView
+
+class HowManyBroughtIntoUkFromSmallProducersViewSpec extends ViewSpecHelper with LitresSpecHelper {
+
+  val howManyBroughtIntoTheUKFromSmallProducersView: HowManyBroughtIntoTheUKFromSmallProducersView =
+    application.injector.instanceOf[HowManyBroughtIntoTheUKFromSmallProducersView]
+
+  implicit val request: Request[_] = FakeRequest()
+  implicit val config: FrontendAppConfig = application.injector.instanceOf[FrontendAppConfig]
+
+  val formProvider = new HowManyAsAContractPackerFormProvider()
```

Comment:
This is the wrong form provider - I think if you change it to the right one, the coverage might be able to get a bit higher (maybe 0.4%)